pkg/logrus/proxy.go

```diff
@@ -3,22 +3,10 @@ package logrus
 import (
 	"context"
 	"fmt"
+	"io"
 	"log/slog"
 	"strings"
-)
-
-type Fields map[string]any
-
-type Level uint32
-
-const (
-	PanicLevel Level = iota
-	FatalLevel
-	ErrorLevel
-	WarnLevel
-	InfoLevel
-	DebugLevel
-	TraceLevel
+	"sync/atomic"
 )
```

Comment:
Types and constants moved to `interface.go`. This now uses `atomic.Pointer` for thread-safe initialization.